spot-oa/api/resources/proxy.py

```diff
@@ -217,23 +212,30 @@ def create_storyboard(uri,date,title,text,expanded_search,top_results):
 --------------------------------------------------------------------------
 """
 def create_timeline(anchor,clientips,date,top_results):
-
     response = ""
     susp_ips = []
 
-
     if clientips:
-	srtlist = sorted(list(clientips.items()), key=lambda x: x[1], reverse=True)
+	    srtlist = sorted(list(clientips.items()), key=lambda x: x[1], reverse=True)
```

Comment:
Is this indentation correct? If yes, is line 220 (for val in strlist[:top_results]: indentation correct?